### PR TITLE
Maint Loot Improvements + Tweaks

### PR DESCRIPTION
**Pull request by BurgerLUA**
Maintenance loot should have more variety now. A lot of the items that appeared way too much for what it was was rebalanced to make it appear less. There is also extra junk now and some more lists. I would list them all but you can just check "changes" for it.

Notable additions include the rare possibility of getting a chameleon item.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by BurgerLUA** [comment]
completely forgot about that lmao

File: code/game/objects/random/random.dm [hunk 1]
```diff
@@ -928,6 +998,17 @@
...
+		/obj/item/weapon/flame/lighter/random = 0.25,
+		/obj/item/weapon/lipstick/random = 0.25,
+		/obj/item/weapon/reagent_containers/glass/beaker/vial/random = 0.1,
+		/obj/item/weapon/towel/random = 0.1,
+		/obj/item/device/flashlight/flare/glowstick/random = 0.25,
+		/obj/item/stack/cable_coil/random/ = 0.1,
+		/obj/item/weapon/bananapeel = 0.1, //honk,
+		/obj/item/weapon/key = 0.1,
+		/obj/item/weapon/reagent_containers/blood/ripped = 0.1,
+		/obj/item/weapon/shreddedp/ = 0.1,
```
Comment (Arrow768) [inline code/game/objects/random/random.dm]: Trailing /

Comment (Alberyk) [inline code/game/objects/random/random.dm]:
```suggestion
		/obj/item/weapon/shreddedp = 0.1
```

**Pull request merged**